### DISCUSSION TITLE
Which light theme do you prefer?

### DISCUSSION THREAD

**Discussion by rustdesk** [rustdesk/rustdesk#3484] [Polls]
Only very little difference.

![image](https://user-images.githubusercontent.com/71636191/222680273-bc95f659-2bfa-4a74-adf7-b06c05172ca0.png)

![image](https://user-images.githubusercontent.com/71636191/222680297-754c79ea-f5d2-4af9-81f9-00b9a38cc22f.png)

**Comment by marcoczen** [comment]
I generally only want dark themes just to reduce eye strain ...

**Reply by rustdesk** [reply]
Agree, our dark theme looks better than light.

**Comment by drynooo** [comment]
Can the delay be lowered further?

**Reply by rustdesk** [reply]
Sure, we will work hard on it.
